src/types/result.rs

```diff
@@ -96,6 +96,8 @@ pub struct CheckResult {
 
     /// Information about the check request made. Will be empty if the check was missed
     pub request_info: Option<RequestInfo>,
+
+    pub region: String,
```

Comment:
Might as well keep up the comments
```suggestion
    /// Region slug that produced the check result
    pub region: String,
```